tests/RuleFilterParser.Tests/LinqTests.cs

```diff
@@ -209,6 +279,32 @@ select new
         //Assert.Equal(3, persons.Count);
         Assert.Equal(5, persons.Count);
     }
+    
+    [Fact(Skip = "Does not work, see https://github.com/bcc-code/rule-filter-parser-dotnet/issues/26")]
+    public async void WhereSelectOrAsyncTest()
```

Comment:
As above. According to #26 this should work.

Reply:
Same as above, works now.